web/src/OverviewActionBar.tsx

```diff
@@ -659,6 +660,88 @@ export function OverviewWidgets(props: { buttons?: UIButton[] }) {
     </WidgetRoot>
   )
 }
+const EnableDisableResourceRoot = styled(ButtonRoot)<{
+  confirmingDisable: boolean
+}>`
+  ${(props) =>
+    props.confirmingDisable &&
+    `
+    color: ${Color.white};
+    background-color: ${Color.red};
+
+    &:hover {
+      color: ${Color.white};
+      background-color: ${Color.red};
```

Comment:
(design nitpick) we may want to choose a different `border-color` for the `confirmingDisable` state in hover/active/non-hover, since the light gray (non-hover) and the blue (hover) clash with the red button. i think making the border red on regular state and darker red on hover/active states would work?

this is _very_ nitpicky, so feel free to not address it now or bring it up in a design review with @SurbhiSGupta!

Reply:
It does look ugly. I'll just disable the hover behavior for now.

Logged this in shortcut [here](https://app.clubhouse.io/windmill/story/12695/resolve-design-questions?cf_workflow=500000005&ct_workflow=all&vc_group_by=day#activity-12700).

Reply:
actually, border-color: white seems reasonable